src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationSubmittedNotification.java

```diff
@@ -49,16 +55,57 @@ public void sendToApplicant2(final CaseData caseData, final Long id) {
             notificationService.sendEmail(
                 caseData.getApplicant2EmailAddress(),
                 JOINT_APPLICATION_SUBMITTED,
-                templateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1()),
+                citizenTemplateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1()),
                 caseData.getApplicant1().getLanguagePreference()
             );
         }
     }
 
-    private Map<String, String> templateVars(CaseData caseData, Long id, Applicant applicant, Applicant partner) {
+    @Override
+    public void sendToApplicant1Solicitor(final CaseData caseData, final Long caseId) {
+
+        if (!caseData.getApplicationType().isSole()) {
+            log.info("Sending joint application submitted notification to applicant 1 solicitor for case : {}", caseId);
+
+            notificationService.sendEmail(
+                caseData.getApplicant1().getSolicitor().getEmail(),
+                SOLICITOR_JOINT_APPLICATION_SUBMITTED,
+                solicitorTemplateVars(caseData, caseId, caseData.getApplicant1()),
+                caseData.getApplicant1().getLanguagePreference()
+            );
+        }
+    }
+
+    @Override
+    public void sendToApplicant2Solicitor(final CaseData caseData, final Long caseId) {
+
+        if (!caseData.getApplicationType().isSole()) {
+            log.info("Sending joint application submitted notification to applicant 2 solicitor for case : {}", caseId);
+
+            notificationService.sendEmail(
+                caseData.getApplicant2().getSolicitor().getEmail(),
```

Comment:
what if it is a paper case joint application and app2 solicitor email is not present ?

Reply:
Adding comment here for traceability. Discussed with BA and as long as an email is provided for a solicitor, they should get the notification (including paper cases). Added a blank/null check to safe guard against empty and null solicitor emails.